templates/search/blocks/search_results__list-card--grid.html

```diff
@@ -30,88 +30,26 @@ <h4 class="tna-heading-s">
             <a href="{% if form.group.value == bucketkeys.ARCHIVE.value %}{% record_url record level_or_archive='Archive' form_group=form.group.value %}{% else %}{% record_url record level_or_archive=record.level form_group=form.group.value %}{% endif %}" class="search-results__list-card-link" data-link="{{ record.reference_prefixed_summary_title }}" data-link-type="Search results list" search-bucket="{{ buckets.current.label }}">
                 {% if record.reference_number %}<span class="sr-only">{{ record.reference_number }}</span>{% endif %}
                 {{ record.summary_title }}
+
+                {# API data required before this can be properly connected to it #}
+                (Type)
             </a>
         </h4>
         <p class="search-results__list-card-description">
             {{ record.listing_description }}
         </p>
-        <dl class="tna-dl tna-dl--plain tna-dl--icon-padding tna-!--margin-bottom-s">
-            {% if record.held_by %}
-                <dt>
-                    <i class="fa-solid fa-landmark"></i>
-                    Held by
-                </dt>
-                <dd>{{ record.held_by }}</dd>
-            {% endif %}
-
+        <dl class="tna-dl tna-dl--plain tna-!--margin-bottom-s">
             {% if record.date_created %}
-                <dt>
-                    <i class="fa-solid fa-calendar"></i>
-                    Date
-                </dt>
+                <dt>Date</dt>
                 <dd>
                     <date>{{ record.date_created }}</date>
                 </dd>
             {% endif %}
 
-            {% if record.reference_number %}
-            <dt>
-                <i class="fa-solid fa-database"></i>
-                {% if form.group.value == 'archive' %}
-                Archon code
-                {% else %}
-                Reference
-                {% endif %}
-            </dt>
-            <dd>{{ record.reference_number }}</dd>
-            {% endif %}
-
-            <!--
-            {% if record.record_opening %}
-                <dt>
-                    Record opening date:
-                </dt>
-                <dd>{{ record.record_opening }}</dd>
-            {% endif %}
-            -->
-            {% if form.group.value == bucketkeys.CREATOR.value and record.type %}
-                <dt>
-                    Type
-                </dt>
-                <dd>{{ record.type }}</dd>
-            {% endif %}
-
-            {% comment %}Debug attributes. Output to allow k-int to adjust the ES index{% endcomment %}
-            <!--
-            <dt>
-                IAID:
-            </dt>
-
-            <dd>{{ record.iaid }}</dd>>
-            <dt>
-                Score:
-            </dt>
-            <dd;>{{ record.score }}</dd>
-            -->
-            {% comment %}End debug attributes.{% endcomment %}
```

Comment:
same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets